LibsAndSamples.sln

```diff
@@ -205,7 +205,6 @@ Global
 		{8D14A8FF-6CB9-44B1-B7C4-3C1D79CE8412}.Debug|ARM64.ActiveCfg = Debug|Any CPU
 		{8D14A8FF-6CB9-44B1-B7C4-3C1D79CE8412}.Debug|ARM64.Build.0 = Debug|Any CPU
 		{8D14A8FF-6CB9-44B1-B7C4-3C1D79CE8412}.Debug|iPhone.ActiveCfg = Debug|Any CPU
-		{8D14A8FF-6CB9-44B1-B7C4-3C1D79CE8412}.Debug|iPhone.Build.0 = Debug|Any CPU
```

Comment:
Did you mean to make these changes to the sln config?

Reply:
no, removing